docs/analyzer/adding_recognizers.md

```diff
@@ -0,0 +1,247 @@
+# Supporting new types of PII entities
+
+Presidio can be extended to support new types of PII entities, and to support additional languages.
```

Comment:
support detection of new types of PII entities (also in the title)

Reply:
added some github "suggestions" for this comment so it is just a matter of click to add if Omri agrees to this